catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/criteria/temporal/TemporalEvaluationCriteriaImpl.java

```diff
@@ -25,21 +25,36 @@
 
     public TemporalEvaluationCriteriaImpl(Date end, Date start, Date input) {
         super();
-        this.end = end;
-        this.start = start;
-        this.input = input;
+        if(end != null) {
```

Comment:
Just out of curiosity. Is it valid for any one of the parameters to be `null`? If not, shouldn't we just use `Validator.notNull()` and fail instead of having all those null checks?